app/templates/application.hbs

```diff
@@ -1,13 +1,17 @@
-{{page-title "My"}}
+{{page-title "My | Real Dev Squad"}}
 
-<Navbar @firstName={{@model.firstName}} @profilePictureURL={{@model.profilePictureURL}} @signOutHandler={{this.signOutHandler}} />
+<Navbar
+  @firstName={{@model.firstName}}
+  @profilePictureURL={{@model.profilePictureURL}}
+  @signOutHandler={{this.signOutHandler}}
+/>
 
 <div class="main-container">
   {{outlet}}
 </div>
-
+<Holder />
```

Comment:
Is this holder component needed here?

Reply:
No it's not. It was an oversight from my part. Will keep it in mind from next time